ledger/participant-integration-api/src/main/scala/platform/localstore/PersistentPartyRecordStore.scala

```diff
@@ -98,6 +98,9 @@ class PersistentPartyRecordStore(
                 _ <- withoutPartyRecord(party) {
                   val newPartyRecord = PartyRecord(
                     party = party,
+                    identityProviderId = partyRecordUpdate.identityProviderIdUpdate.getOrElse(
+                      IdentityProviderId.Default
```

Comment:
This value needs to be eventually replaced with the current authorized user's idp id, correct?
If so, please add a TODO note

Reply:
This will stay as `Default` as here we get an Update from the API and id might not be set there.

Reply:
What if I'm a user in idp_foo and issue an update user rpc without specifying any idp id. Will the idp of the updated user be reset to default?

Reply:
let's take this discussion offline and discuss it, it's a bit more complicated than I thought